.github/workflows/tests-integration.yml

```diff
@@ -0,0 +1,22 @@
+name: Run Unit Tests
+on:
+  workflow_call
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
Can you confirm this must be Java 11 whereas the others are OK to be Java 14

Reply:
Yes. I can confirm that `mvn test` works fine with Java 14. But -- `mvn verify` needs Java 11.